lib/puppet_x/sqlserver/sql_connection.rb

```diff
@@ -9,7 +9,7 @@ def open_and_run_command(query, config)
           open(config)
           execute(query)
         rescue win32_exception => e
```

Comment:
is `=> e` still necessary if you're not going to do anything with the specific exception?

Reply:
Probably not. I'll get rid of it.

Reply:
Hey, turns out I do still need it.

Reply:
General rubyism is to prefix with underscore for required, but unused variables  `_e`